changelog-maker.js

```diff
@@ -11,6 +11,7 @@ const spawn    = require('child_process').spawn
     , ghissues = require('ghissues')
     , chalk    = require('chalk')
     , pkgtoId  = require('pkg-to-id')
+    , mdExtract= require('markdown-extract')
```

Comment:
just push all the `=`'s out rather than squashing up like this, also feel free to spell it out fully if you'd rather, like `commitStream` which is long enough to justify not lining up `=`, just make it so that you don't have the `=` up against the variable name